chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -3358,4 +3370,41 @@ public void setAnimationTime(double animationTime) {
   public double getAnimationTime() {
     return animationTime;
   }
+
+  /**
+   * Add additional data
+   * Additional data is not used by chunky but can be used by plugins
+   */
+  @PluginApi
+  public void setData(String name, JsonValue value) {
+    additionalData.add(name, value);
+  }
+
+  /**
+   * Retrieve additional data
+   * Additional data is not used by chunky but can be used by plugins
+   */
+  @PluginApi
+  public JsonValue getData(String name) {
+    return additionalData.get(name);
+  }
+
+  /**
+   * Get the whole additional data object
+   * Additional data is not used by chunky but can be used by plugins
+   */
+  @PluginApi
+  public JsonObject getAdditionalData() {
+    return additionalData;
+  }
+
+  /**
+   * Sets the whole additional data object
+   * Additional data is not used by chunky but can be used by plugins
+   * Prefer using `setData` to not remove the additional of another plugin
```

Comment:
Oh, I'm sorry, my comment was bad. I didn't mean to add methods to get/set the whole object but just rename `getData` to something that makes clear what "data" is instead. Not sure if replacing the entire data is useful to plugins… maybe it is?

```suggestion
   * Prefer using {@link #setData} to not remove the additional of another plugin
```